ERCS/erc-7556.md

```diff
@@ -0,0 +1,157 @@
+---
+eip: 7556
+title: Embedded Accounts as Smart Modules
+description: TODO
```

Comment:
You'll need something here, even if it isn't your final description.

For inspiration: What's an embedded account? Smart module? Who might want to use this proposal?